src/data-tutorials/users.tsx

```diff
@@ -2726,6 +2726,22 @@ const Users: User[] = [
     website: 'blog/tutorials/building-a-celo-voting-system-with-golang',
     tags: ['celosage', 'intermediate', 'smartcontract', 'solidity'], 
 
+  },
+   {
+    title: 'Deploying Celo Applications using Golang',
```

Comment:
Can you correct this also, your user.tsx file

Reply:
Alright I'll make the changes